### ISSUE TITLE
Cannot import modules/table. Are you sure it was registered? 

### ISSUE THREAD

**Issue by MeXaaR** [volser/quill-table-ui#6]
Hi guys, I have a problem on the module registration with React Quill

I do this

```import * as QuillTableUI from 'quill-table-ui'
import ReactQuill, { Quill } from 'react-quill';

Quill.register({
  'modules/tableUI': QuillTableUI
}, true)

//component 
<ReactQuill
   id="content" 
   value={content} 
   onChange={onUpdateRichText} 
  modules={{
    table: true,
    tableUI: true,
    ...otherModules
}}
/>
```

And I've got a big red error
`quill Cannot import modules/table. Are you sure it was registered? `

I tried different thing but no success. Do you have a tip for me to register table module please ?

**Comment by shengslogar** [comment]
The tables module is only available on the Quill 2.0 dev version and hasn't made it to the stable channel yet. `npm install quill@2.0.0-dev.4` is the latest.

**Comment by rimzzlabs** [comment]
> The tables module is only available on the Quill 2.0 dev version and hasn't made it to the stable channel yet. `npm install quill@2.0.0-dev.4` is the latest.

I've upgraded my package, I still got the error

**Comment by SimeonKovachev** [comment]
How did you solve this?

**Comment by rimzzlabs** [comment]
> How did you solve this?

I don't think it is solved, I recommend to check other library such as Tip Tap or Lexical
